src/sagemaker/deserializers.py

```diff
@@ -55,17 +56,44 @@ def ACCEPT(self):
         """The content types that are expected from the inference endpoint."""
 
 
-class StringDeserializer(BaseDeserializer):
-    """Deserialize data from an inference endpoint into a decoded string."""
+class SimpleBaseDeserializer(with_metaclass(abc.ABCMeta, BaseDeserializer)):
+    """Abstract base class for creation of new deserializers.
+
+    This class extends the API of :class:~`sagemaker.deserializers.BaseDeserializer` with more
+    user-friendly options for setting the ACCEPT content type header, in situations where it can be
+    provided at init and freely updated.
+    """
+
+    def __init__(self, accept="*/*"):
+        """Initialize a ``SimpleBaseDeserializer`` instance.
+
+        Args:
+            accept (union[str, tuple[str]]): The MIME type (or tuple of allowable MIME types) that
+                is expected from the inference endpoint (default: "*/*").
+        """
+        super(SimpleBaseDeserializer, self).__init__()
+        self.accept = accept
+
+    @property
+    def ACCEPT(self):
+        """The tuple of possible content types that are expected from the inference endpoint."""
```

Comment:
```suggestion
        """Content type or the tuple of possible content types that are expected from the inference endpoint.
            
            Returns: MIME type or tuple of MIME types.
        """
```

Reply:
But in the current implementation, `ACCEPT` property always returns a tuple, right? It's the`.accept` instance variable that could be str or Tuple[str]?